dev-requirements.txt

```diff
@@ -45,6 +46,7 @@ sortedcontainers==2.3.0
 stevedore==3.3.0; python_version >= '3.6'
 toml==0.10.2; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'
 typed-ast==1.4.2
-typing-extensions==3.7.4.3
+typing-extensions==3.7.4.3; python_version < '3.8' and python_version < '3.8'
```

Comment:
@derekpierre Can you describe what version of python and pipenv are you using? I noticed some differences in the locking process depending on the environment. See 
https://discord.com/channels/411401661714792449/411401661714792451/796211275411619871

Reply:
@cygnusv `pip 20.3.3` and `Python 3.7.7`.